app/models/checklist_test.rb

```diff
@@ -51,6 +51,14 @@ def measure_status(measure_id)
     pass_count == criterias.count ? 'passed' : 'failed'
   end
 
+  def negate_valueset?(measure, criteria_key)
+    if measure.hqmf_document[:data_criteria].select { |key| key == criteria_key }.values.first.negation
+      [true, false].sample
```

Comment:
after sponsor feedback.  Let's always have this return true instead of randomly true or false.